docs/configuration.md

```diff
@@ -550,6 +550,8 @@ print(config2.two) # => 102
 To define a secret for your BigFlow workflow, you need to use [Kubernetes Secrets](https://cloud.google.com/composer/docs/how-to/using/using-kubernetes-pod-operator#secret-config)
 mechanism (Cloud Composer is a GKE cluster under the hood).
 
+**Secrets management based on Kubernetes Secrets works only if a job is scheduled in Composer. When you want to run the job from a local machine secrets should be provided as environment variables.**
```

Comment:
Is it specified somewhere else what are these env variables?

Reply:
Environment variables on your computer ;)

Reply:
I added that

Reply:
I know, but what variables (names) do I have to set?